mutating-admission-controller/Makefile

```diff
@@ -0,0 +1,15 @@
+LDFLAGS := "-extldflags '-static'"
+
+MOD ?= vendor
+
+.PHONY: install-slim
+# Once we change CI code to build outside GOPATH, GO111MODULE can be removed,
+# so we rely on defaults.
+install-slim:
```

Comment:
I'd change this target name to just `all` , which is a standarized target for building the binary: https://www.gnu.org/prep/standards/html_node/Standard-Targets.html.

`install` might be counter-intuitive, as it usually copies built files into host system, which is usually unwanted while developing.